gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java

```diff
@@ -393,12 +393,12 @@ public boolean isVisible() {
 
 							addInstructorAttributeOrHide(sakaiRubricButton, assignment, studentUuid, showingStudentView);
 
-							Optional<AssociationTransferBean> optAssociation
```

Comment:
As you're already modifying existing lines to use this format, there is another place just a few lines before.